setup.cfg

```
@@ -20,6 +20,7 @@ classifiers =
     Operating System :: MacOS :: MacOS X
     Operating System :: Microsoft :: Windows
     Operating System :: POSIX :: Linux
+    Operating System :: POSIX :: Linux-ROOT
     Operating System :: Unix
```

setup.cfg is not involved in configuring CI. This list here is just the list of OSes that Awkward is known to run on, so "Linux-ROOT" is not instance to be listed.

```suggestion
    Operating System :: POSIX :: Linux
    Operating System :: Unix
```